Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/Views.kt

```diff
@@ -54,3 +54,27 @@ fun TextView.setUrl(@StringRes textRes: Int, label: String, url: String) {
         }
     }
 }
+
+fun TextView.setUrl(@StringRes textRes: Int, @StringRes labelRes: Int, @StringRes urlRes: Int) {
```

Comment:
Why have you added this function, wasn't it already in place?

Reply:
Previous function used strings as label and url. So I added new function to deal with all resources.

Reply:
This function is doing the exact same thing as the one above. So I think it makes sense to just call it after you retrieved the strings for `url` and `label`

Reply:
As @mtwalli suggested below I refactor it to use Core-ktx buildSpannedString. Kept other functions untouched to keep it safe.